operations/messaging-and-math/how-to-guides-for-m-and-m/how-to-create-release-announcements.md

```diff
@@ -107,6 +109,43 @@ The Release Manager \(Amy Blais\) owns the release announcement, including the f
 4. Do not use internal names of features, such as "Unread toasts", "Deep linking"
 5. Say "You" instead of "Users" to talk directly to the reader
 
+## Scoring Guidelines
```

Comment:
For the scoring guidelines, can you help review them and ensure they follow our 3C's of communication:
- they are clear (opportunity: read from the perspective of someone who is not familiar with our release announces)
- they are concise (I think so yes)
- they are consistent (opportunity: is the scoring consistent with the guidelines we have for title and intro bullets above)